### PR TITLE
Add TCK results for MP Config 2.0, MP Fault Tolerance 3.0 and MP Open API 2.0

### PR DESCRIPTION
**Pull request by tevans78**
Part of MicroProfile 4.0 certification

### CODE REVIEW

**Review by mbroz2 — COMMENTED** [review]
is there a reason we're versioning microprofile in the URL? microprofile/4.0/config/2.0/TCKResults.adoc

I would prefer microprofile to be unversioned, and just version the subspec, so
microprofile/config/2.0/TCKResults.adoc
microprofile/faulttolerance/3.0/TCKResults.adoc
microprofile/openapi/2.0/TCKResults.adoc

**Review by mbroz2 — COMMENTED** [review]
Also, can you drop the .gitignore from the PR

File: microprofile/4.0/config/2.0/TCKResults.adoc [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+  <testsuite errors="0" failures="0" hostname="ebcprh0937mcdfd9e-n.fyre.ibm.com" id="32" name="TestConfigProfileTest" package="org.eclipse.microprofile.config.tck.profile" tests="1" time="0.296" timestamp="17 Mar 2021 08:26:54 GMT">
+      <testcase classname="org.eclipse.microprofile.config.tck.profile.TestConfigProfileTest" name="testConfigProfileWithDev_EE8_FEATURES" time="0.296" />
+  </testsuite>
+  <testsuite errors="0" failures="0" hostname="ebcprh0937mcdfd9e-n.fyre.ibm.com" id="33" name="TestCustomConfigProfile" package="org.eclipse.microprofile.config.tck.profile" tests="1" time="0.366" timestamp="17 Mar 2021 08:26:54 GMT">
+      <testcase classname="org.eclipse.microprofile.config.tck.profile.TestCustomConfigProfile" name="testConfigProfileWithDev_EE8_FEATURES" time="0.366" />
+  </testsuite>
+</testsuites>
+
+
+----
```
Comment (Emily-Jiang) [inline microprofile/4.0/config/2.0/TCKResults.adoc:922]: Please also add Java 11 test result.

File: microprofile/4.0/faulttolerance/3.0/TCKResults.adoc [hunk 2]
```diff
@@ -0,0 +1,699 @@
...
+      <testcase classname="org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest" name="serviceBaseROCMNoRedefinition" time="0.342" />
+      <testcase classname="org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest" name="serviceBaseROCMOverridedClassLevelNoMethodOverride" time="0.366" />
+      <testcase classname="org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest" name="serviceBaseROMOverridedMethodLevel" time="0.499" />
+      <testcase classname="org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest" name="baseRetryServiceUsesDefaults" time="0.731" />
+      <testcase classname="org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest" name="serviceBaseROCM" time="0.196" />
+      <testcase classname="org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest" name="serviceRetryRemovedAtMethodLevel" time="0.569" />
+  </testsuite>
+</testsuites>
+----
```
Comment (Emily-Jiang) [inline microprofile/4.0/faulttolerance/3.0/TCKResults.adoc:1372]: Please add Java 11 test result as well

File: microprofile/4.0/openapi/2.0/TCKResults.adoc [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+      <testcase classname="org.eclipse.microprofile.openapi.tck.PetStoreAppTest" name="testDefaultResponseType" time="0.059" />
+      <testcase classname="org.eclipse.microprofile.openapi.tck.PetStoreAppTest" name="testAPIResponseSchema" time="0.413" />
+      <testcase classname="org.eclipse.microprofile.openapi.tck.PetStoreAppTest" name="testSecurityScheme" time="0.213" />
+  </testsuite>
+  <testsuite errors="0" failures="0" hostname="ebcprh0935mcdesu3-n.fyre.ibm.com" id="17" name="StaticDocumentTest" package="org.eclipse.microprofile.openapi.tck" skipped="0" tests="2" time="2.324" timestamp="17 Mar 2021 08:33:40 GMT">
+      <testcase classname="org.eclipse.microprofile.openapi.tck.StaticDocumentTest" name="testStaticDocument" time="1.290" />
+      <testcase classname="org.eclipse.microprofile.openapi.tck.StaticDocumentTest" name="testStaticDocument" time="1.034" />
+  </testsuite>
+</testsuites>
+----
```
Comment (Emily-Jiang) [inline microprofile/4.0/openapi/2.0/TCKResults.adoc:596]: Please add Java 11 test result as well.

File: microprofile/4.0/config/2.0/TCKResults.adoc (hunk 1 above)
Comment (tevans78) [reply microprofile/4.0/config/2.0/TCKResults.adoc:922]: These results are Java SE 11 running with Java EE 8

Comment (tevans78) [reply microprofile/4.0/config/2.0/TCKResults.adoc:922]: Java 8 results added

File: microprofile/4.0/faulttolerance/3.0/TCKResults.adoc (hunk 2 above)
Comment (tevans78) [reply microprofile/4.0/faulttolerance/3.0/TCKResults.adoc:1372]: Java 8 results added

File: microprofile/4.0/openapi/2.0/TCKResults.adoc (hunk 3 above)
Comment (tevans78) [reply microprofile/4.0/openapi/2.0/TCKResults.adoc:596]: Java 8 results added

**Review by Emily-Jiang — APPROVED** [review]
LGTM! Thank you @tevans78 !

**Review by mbroz2 — APPROVED** [review]

**Pull request merged**